Main.java

```diff
@@ -2,12 +2,12 @@ class Main {
 	public static void main(String[] args)
 	{
 		if (args.length > 0) {
-			System.out.println(" " + " Argument");
+			System.out.println("Argument");
 			for (String val : args)
```

Comment:
Nice job until now, the only thing that is still missing is, that you should not print ALL arguments, you should only print the FIRST argument.